Webhook Payload Error / Failure

I'm using version v0.10.1 on Windows Server 2022 and everything is working well, except for sending a webhook to Slack.  Sending the webhook always fails, with the following error in the logs -
```
[ERROR] Webhook Payload (E:\Torrents\Sonarr\Some.File.Name = extracted): template: webhook:64:15: executing "webhook" at <.Data>: Data has arguments but cannot be invoked as function
```

Here is my config for the webhook -

```ApacheConf
################
### Webhooks ###
################
# Sends a webhook when an extraction queues, starts, finishes, and/or is deleted.
# Created to integrate with notifiarr.com.
# Also works natively with Discord.com, Telegram.org, and Slack.com webhooks.
# Can possibly be used with other services by providing a custom template_path.
###### Don't forget to uncomment [[webhook]] and url at a minimum !!!!
[[webhook]]
url    = "https://hooks.slack.com/services/xxxxx/xxxxxxxxxx/xxxxxxxxxxxxxxxxx"
name   = "Slack"    # Set this to hide the URL in logs.
silent = true # do not log success (less log spam)
events = [0]   # list of event ids to include, 0 == all.
## Advanced Optional Webhook Configuration
nickname      = "Unpackerr"    # Used in Discord and Slack templates as bot name, in Telegram as chat_id.
#  channel       = "unpackerr"    # Also passed into templates. Used in Slack templates for destination channel.
#  exclude       = []    # list of apps to exclude, ie. ["radarr", "lidarr"]
#  template_path = ""    # Override internal webhook template for discord.com or other hooks.
#  template      = ""    # Override automatic template detection. Values: notifiarr, discord, telegram, gotify, pushover, slack
#  ignore_ssl    = false # Set this to true to ignore the SSL certificate on the server.
#  timeout       = "10s" # You can adjust how long to wait for a server response.
#  content_type  = "application/json" # If your custom template uses another MIME type, set this.
```

Let me know if I can provide any additional information or test anything!

--- a/pkg/unpackerr/webhooks_templates.go
+++ b/pkg/unpackerr/webhooks_templates.go
@@ -234,7 +234,7 @@ const WebhookTemplateSlack = `
           "text": "*Elapsed*\n{{.Data.Elapsed}}"
         }{{end}}{{end -}}
       ]
-    }{{if and (.Data .Data.Error)}},{
+    }{{if and .Data .Data.Error}},{
       "type": "section",
       "text": {
         "type": "mrkdwn",

